dev/com.ibm.ws.cdi.lifecycle_fat/test-applications/ObservesInitializedInJars.war/src/cdi12/observersinjarsbeforebean/WarBeforeBeansObserver.java

```diff
@@ -1,60 +0,0 @@
-/*******************************************************************************
- * Copyright (c) 2017 IBM Corporation and others.
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v1.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-v10.html
- *
- * Contributors:
- *     IBM Corporation - initial API and implementation
- *******************************************************************************/
-package cdi12.observersinjarsbeforebean;
-
-import javax.enterprise.event.Observes;
-import javax.enterprise.inject.spi.BeforeBeanDiscovery;
-import javax.enterprise.inject.spi.Extension;
-
-import cdi12.observersinjars.SomeClass;
-
-public class WarBeforeBeansObserver implements Extension {
-
-    private static String beforeBeanDiscoveryCalled = "Method not called";
-
-    private ClassLoader TCCL = null;
-    private ClassLoader myCL = null;
```

Comment:
This class is a bit odd.

It was registered as an extension and `beforeBeanDiscovery` logs whether certain classes are loadable from the app class loader and the TCCL.

However, nothing was asserting what the values should be. I took it out to simplify the test but if we decide that we do want to assert something here we could add it back in.